MySQL/system/ksp-audit-cis-mysql-1-1.yaml

```diff
@@ -0,0 +1,21 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-audit-cis-mysql-1-1
+  namespace: default
+spec:
+  tags: ["MySQL", "CIS", "rule-1-1"]
+  message: "Notification! mysqld service accessed!"
+  selector:
+    matchLabels:
+      app: mysql
+  file:
+    severity: 1
+    matchDirectories:
+    - dir: /var/lib/mysql/
+    - dir: /media/mysql/
+    action: Audit
```

Comment:
Cannot audit these paths. mysql bin will use this path to keep its data and will result in alert deluge.